reflex/vars.py

```diff
@@ -685,6 +685,74 @@ def __ror__(self, other: Var) -> Var:
         """
         return self.operation("||", other, type_=bool, flip=True)
 
+    def __contains__(self, _: Any) -> Var:
+        """Override the 'in' operator to alert the user that it is not supported.
+
+        Raises:
+            TypeError: the operation is not supported
+        """
+        raise TypeError(
+            "'in' operator not supported for Var types, use Var.contains() instead."
```

Comment:
so the return of `__contains__` seems to be automatically converted to boolean, so instead of something like a var with `{[1,2,3].includes(1)}`, we get `True` (the var being eval as a boolean).